src/doc/guide.md

```diff
@@ -419,6 +419,17 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+
+## Build cache
+
+Cargo shares build artifacts among all the packages of a single workspaces.
+If you want to share build results across different workspaces, you can use
```

Comment:
Perhaps this could explicitly call out that this feature isn't built-in to cargo today but sccache, a third party tool, can be used to achieve similar results?